implementations/rust/ockam/ockam_api/README.md

```diff
@@ -11,6 +11,123 @@ and trustfully with cloud services and other devices.
+│  ├─ identity1.json
+│  ├─ identity2.json
+│  └─ ...
+├─ nodes
+│  ├─ node1
+│  │  ├─ default_identity -> ...
+│  │  ├─ default_vault -> ...
+│  │  ├─ policies-storage.lmdb
+│  │  ├─ policies-storage.lmdb-lock
+│  │  ├─ setup.json
+│  │  ├─ stderr.log
+│  │  ├─ stdout.log
+│  │  └─ version
+│  ├─ node2
+│  └─ ...
+├─ projects
+│  └─ default.json
+├─ trust_contexts
+│  └─ default.json
+└─ vaults
+   ├─ vault1.json
+   ├─ vault2.json
+   ├─ ...
+   └─ data
+      ├─ vault1.lmdb
+      ├─ vault1.lmdb-lock
+      ├─ vault2.lmdb
+      ├─ vault2.lmdb-lock
+      └─ ...
+```
+## `credentials`
+
+Each file stored under the `credentials` directory contains the credential for a given identity.
+Those files are created with the `ockam credential store` command. They are then read during the creation of
+a secure channel to send the credentials to the other party
+
+## `defaults`
+
+This directory contains symlinks to other files or directories in order to specify which node,
+identity, credential or vault must be considered as a default when running a command expecting those
+inputs
+
+## `identities`
+
+This directory contains one file per identity and a data directory. An identity file is created
+with the `ockam identity create` command or created by default for some commands (in that case the
+`defaults/identity` symlink points to that identity). The identity file contains:
+
+- the identity identifier
+- the enrollment status for that identity
+
+The `data` directory contains a LMDB database with other information about identities:
+ - the credential attributes that have been verified for this identity. Those attributes are
+   generally used in ABAC rules that are specified on secure channels. For example when sending messages
+   via a secure channel and using the Orchestrator the `project` attribute will be checked and the LMDB database accessed
+ - the list of key changes for each identity. This information is accessed in order to get the latest public key
+   associated to a given identity when checking its signature during the creation of a secure channel.
+   It is also accessed to retrieve the key id associated to that key and then use a Vault to create a signature
+   for an identity
+
+Note: for each `.lmdb` file there is a corresponding `lmdb-lock` file which is used to control
+the exclusive access to the LMDB database even if several OS processes are trying to modify it.
+
+## `nodes`
+
+This directory contains:
+
+ - symlinks to default values for the node: identity and vault
+ - a database for ABAC policies
+ - a setup file containing some configuration information for the node (is it an authority node?, what is the TCP listener address?,...)
+ - log files: for system errors and system outputs. The stdout.log file is where almost all the node logs are written
+ - a version number for the configuration
+
+## `projects`
+
+This directory contains a list of files, one per project that was created, either the default project
```

Comment:
If we start a node in the environment which has project configurations, does the node have access to these configurations?

Reply:
When you start a node you either specify nothing and that node uses the default project or you specify one of the project configurations.